<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>設定</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/resources.qrc">
    <normaloff>:/icons/Gear.png</normaloff>:/icons/Gear.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="hWid" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border : 0px</string>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea {border : 0px}</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>544</width>
           <height>436</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="general">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>一般</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTabWidget" name="tabWidget">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="settingTab">
                 <attribute name="title">
                  <string>全般</string>
                 </attribute>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="profile">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>プロファイル</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="addButton_2">
                   <property name="text">
                    <string>追加</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="deleteButton_2">
                   <property name="text">
                    <string>削除</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="copyButton_2">
                   <property name="text">
                    <string>コピー</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="upButton_2">
                   <property name="text">
                    <string>上へ</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="downButton_2">
                   <property name="text">
                    <string>下へ</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="openButton_2">
                   <property name="text">
                    <string>開く</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="ProfileTable" name="profileTableWidget">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="plugin">
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>拡張機能</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTabWidget" name="profileTabWidget_2">
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <widget class="QWidget" name="string">
                 <attribute name="title">
                  <string>Global</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="frame_5">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="addButton_5">
                       <property name="text">
                        <string>追加</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Add.png</normaloff>:/icons/Add.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="deleteButton_5">
                       <property name="text">
                        <string>削除</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Denided.png</normaloff>:/icons/Denided.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="copyButton_5">
                       <property name="text">
                        <string>コピー</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Files_Copy.png</normaloff>:/icons/Files_Copy.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="upButton_5">
                       <property name="text">
                        <string>上へ</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Button_Up.png</normaloff>:/icons/Button_Up.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="downButton_5">
                       <property name="text">
                        <string>下へ</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Button_Down.png</normaloff>:/icons/Button_Down.png</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="StringTable" name="stringTableWidget">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="search">
                 <attribute name="title">
                  <string>検索</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="frame_4">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="addButton_4">
                       <property name="text">
                        <string>追加</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Add.png</normaloff>:/icons/Add.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="deleteButton_4">
                       <property name="text">
                        <string>削除</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Denided.png</normaloff>:/icons/Denided.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="copyButton_4">
                       <property name="text">
                        <string>コピー</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Files_Copy.png</normaloff>:/icons/Files_Copy.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="upButton_4">
                       <property name="text">
                        <string>上へ</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Button_Up.png</normaloff>:/icons/Button_Up.png</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="downButton_4">
                       <property name="text">
                        <string>下へ</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../res/resources.qrc">
                         <normaloff>:/icons/Button_Down.png</normaloff>:/icons/Button_Down.png</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="SearchTable" name="searchTableWidget">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="script">
                 <attribute name="title">
                  <string>dll</string>
                 </attribute>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="frame_6">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="addButton_6">
                       <property name="text">
                        <string>追加</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="copyButton_6">
                       <property name="text">
                        <string>コピー</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="deleteButton_6">
                       <property name="text">
                        <string>削除</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_4">
                       <property name="text">
                        <string>開く</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="ExtraFunctionsTable" name="scriptTableWidget">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StringTable</class>
   <extends>QTableWidget</extends>
   <header>../stringtable/stringtable.h</header>
  </customwidget>
  <customwidget>
   <class>SearchTable</class>
   <extends>QTableWidget</extends>
   <header>../searchtable/searchtable.h</header>
  </customwidget>
  <customwidget>
   <class>ProfileTable</class>
   <extends>QTableWidget</extends>
   <header>profiletable.h</header>
  </customwidget>
  <customwidget>
   <class>ExtraFunctionsTable</class>
   <extends>QTableWidget</extends>
   <header>../extrafunctionstable/extrafunctionstable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../res/resources.qrc"/>
  <include location="../../../res/resources.qrc"/>
  <include location="../../../res/resources.qrc"/>
  <include location="../../../res/resources.qrc"/>
  <include location="../../../res/resources.qrc"/>
  <include location="../../../res/resources.qrc"/>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
